tests/integ/scala/test_dataframe_range_suite.py

```diff
@@ -108,3 +108,15 @@ def test_range_with_max_and_min(session):
     end = MIN_VALUE + 2
     assert session.range(start, end, 1).collect() == []
     assert session.range(start, start, 1).collect() == []
+
+
+@pytest.mark.localtest
+def test_range_with_large_range_and_step(session):
+    try:
```

Comment:
We could also just skip the test with something like https://github.com/snowflakedb/snowpark-python/blob/f47c158bbf8e4e52f43b6ec3d1476be0bbc86095/tests/unit/test_code_generation.py#L548

Reply:
+1